frontend/src/components/Edit.jsx

```diff
@@ -0,0 +1,201 @@
+import React, { Component } from "react";
+import axios from "axios";
+import * as yup from "yup";
+import {
+  DigitText,
+  DigitTextField,
+  DigitTextArea,
+  DigitForm,
+  DigitFormField,
+  DigitDesign,
+  DigitButton
+} from "@cthit/react-digit-components";
+import IngredientCreator from "./elements/upload/IngredientCreator";
+import "./styles/Edit.css";
+
+class Edit extends Component {
+  constructor(props) {
+    super(props);
+    let currentRecipe = JSON.parse(localStorage.getItem("recipeData"));
```

Comment:
Quick question, are you using localStorage instead of backend until the backend is done?